evm/src/executor/mod.rs

```diff
@@ -423,6 +458,7 @@ where
             labels,
             traces,
             debug,
+            transactions,
```

Comment:
should we bundle cheatcode-related state in a struct? labels, traces, transactions